weave-js/src/components/Panel2/PanelString.tsx

```diff
@@ -214,11 +214,29 @@ export const PanelString: React.FC<PanelStringProps> = props => {
       );
     }
 
-    const contentPlaintext = (
-      <S.PreformattedProportionalString>
-        {fullStr}
-      </S.PreformattedProportionalString>
-    );
+    let parsed: any;
+    if (fullStr.startsWith('{') && fullStr.endsWith('}')) {
```

Comment:
Might be useful to trim whitespace from the string before this check?

Reply:
👍